### DISCUSSION TITLE
Dúvida sobre Processo do pagamento Agendado (Reivindicação e Portabilidade)

### DISCUSSION THREAD

**Discussion by LeoBOrtigoza-zz** [bacen/pix-api#380] [Q&A]
Olá @ninrod, boa tarde!! Como está?

Tenho uma dúvida sobre processo. 

Quando eu ler um QR com vencimento e salvar as informações. Caso tenha uma portabilidade ou reivindicação. Como devo tratar aqui como papel de PSP Pagador?

Por que entendo que não devemos consultar novamente a location cobv no pagamento, pq poderia ser uma experiência ruim. Por que o comprovante já foi gerado. 

Seria responsabilidade do PSP Recebedor negar esse pagamento?  Mesmo que um novo PSP receba essa chave futuramente.

**Comment by ninrod** [comment]
@LeoBOrtigoza , boa tarde.

Entendo que no momento em que você guarda as informações a respeito do pagamento, não há mais o que se falar em termos de chave.  O pagamento será realizado na exata conta para a qual a chave estava apontando no momento em que se consultou o DICT.

**Reply by LeoBOrtigoza-zz** [reply]
Olá, boa tarde!!

muito obrigado pelo retorno.

mesmo que o pagamento seja feito 20 dias depois?

**Reply by LeoBOrtigoza-zz** [reply]
por que no caso, eu vou ter a chave, porem, pode ser que as informações na DICT para aquele pagamento seja diferente de quando eu consultei

**Reply by ninrod** [reply]
> mesmo que o pagamento seja feito 20 dias depois?

Sim. 

> por que no caso, eu vou ter a chave, porem, pode ser que as informações na DICT para aquele pagamento seja diferente de quando eu consultei

Vale o que constava na cobrança original. Não existe, em nenhum lugar da documentação do pix, esse processo de você verificar novamente a chave. 

O que pode acontecer, no entanto, e isso é previsto, é o PSP recebedor rejeitar o pagamento na DPP. Isso pode se dar por N motivos, inclusive pelo motivo de a cobrança ter sido removida pelo PSP recebedor, por exemplo.

**Reply by LeoBOrtigoza-zz** [reply]
excelente!! Muito obrigado. 

ai, gerou uma dúvida. Quando consulta a dict eu preciso gerar o idendtoend, nesse caso, eu preciso gerar o idendtoend com o padrão de data do pagamento futuro? Para não ter expiração?

**Reply by ninrod** [reply]
@LeoBOrtigoza , aí com relação ao gerencimento do end2end já foge do fórum aqui. Recomendo perguntar diretamente ao pix-operacional.

**Reply by LeoBOrtigoza-zz** [reply]
claro @ninrod! 

muito obrigado e ótimo dia!!

abraços
